set -e
true
true
/sw/bioinfo/spades/3.15.3/rackham/bin/spades-hammer /proj/uppmax2022-2-10/nobackup/innaf/Genome_analysis_project/corrected/configs/config.info
/usr/bin/python /sw/bioinfo/spades/3.15.3/rackham/share/spades/spades_pipeline/scripts/compress_all.py --input_file /proj/uppmax2022-2-10/nobackup/innaf/Genome_analysis_project/corrected/corrected.yaml --ext_python_modules_home /sw/bioinfo/spades/3.15.3/rackham/share/spades --max_threads 16 --output_dir /proj/uppmax2022-2-10/nobackup/innaf/Genome_analysis_project/corrected --not_used_yaml_file /proj/uppmax2022-2-10/nobackup/innaf/Genome_analysis_project/corrected/dont_correct.yaml --gzip_output
true
true
/sw/bioinfo/spades/3.15.3/rackham/bin/spades-core /proj/uppmax2022-2-10/nobackup/innaf/Genome_analysis_project/K55/configs/config.info
/usr/bin/python /sw/bioinfo/spades/3.15.3/rackham/share/spades/spades_pipeline/scripts/copy_files.py /proj/uppmax2022-2-10/nobackup/innaf/Genome_analysis_project/K55/before_rr.fasta /proj/uppmax2022-2-10/nobackup/innaf/Genome_analysis_project/before_rr.fasta /proj/uppmax2022-2-10/nobackup/innaf/Genome_analysis_project/K55/assembly_graph_after_simplification.gfa /proj/uppmax2022-2-10/nobackup/innaf/Genome_analysis_project/assembly_graph_after_simplification.gfa /proj/uppmax2022-2-10/nobackup/innaf/Genome_analysis_project/K55/final_contigs.fasta /proj/uppmax2022-2-10/nobackup/innaf/Genome_analysis_project/contigs.fasta /proj/uppmax2022-2-10/nobackup/innaf/Genome_analysis_project/K55/first_pe_contigs.fasta /proj/uppmax2022-2-10/nobackup/innaf/Genome_analysis_project/first_pe_contigs.fasta /proj/uppmax2022-2-10/nobackup/innaf/Genome_analysis_project/K55/strain_graph.gfa /proj/uppmax2022-2-10/nobackup/innaf/Genome_analysis_project/strain_graph.gfa /proj/uppmax2022-2-10/nobackup/innaf/Genome_analysis_project/K55/scaffolds.fasta /proj/uppmax2022-2-10/nobackup/innaf/Genome_analysis_project/scaffolds.fasta /proj/uppmax2022-2-10/nobackup/innaf/Genome_analysis_project/K55/scaffolds.paths /proj/uppmax2022-2-10/nobackup/innaf/Genome_analysis_project/scaffolds.paths /proj/uppmax2022-2-10/nobackup/innaf/Genome_analysis_project/K55/assembly_graph_with_scaffolds.gfa /proj/uppmax2022-2-10/nobackup/innaf/Genome_analysis_project/assembly_graph_with_scaffolds.gfa /proj/uppmax2022-2-10/nobackup/innaf/Genome_analysis_project/K55/assembly_graph.fastg /proj/uppmax2022-2-10/nobackup/innaf/Genome_analysis_project/assembly_graph.fastg /proj/uppmax2022-2-10/nobackup/innaf/Genome_analysis_project/K55/final_contigs.paths /proj/uppmax2022-2-10/nobackup/innaf/Genome_analysis_project/contigs.paths
true
/usr/bin/python /sw/bioinfo/spades/3.15.3/rackham/share/spades/spades_pipeline/scripts/breaking_scaffolds_script.py --result_scaffolds_filename /proj/uppmax2022-2-10/nobackup/innaf/Genome_analysis_project/scaffolds.fasta --misc_dir /proj/uppmax2022-2-10/nobackup/innaf/Genome_analysis_project/misc --threshold_for_breaking_scaffolds 3
true
